doc/1.0-roadmap.md

```diff
@@ -0,0 +1,110 @@
+# client-rust 1.0 roadmap
+
+The client-rust project complements the TiKV ecosystem by providing a reliable Rust client. It can be a foundation for implementing clients in other languages.
+
+The document describes our plan to push client-rust towards its general availability (1.0).
+
+## Deliverable
+
+client-rust 1.0 that
+- enables new users to start using it in 10 mins
+- has been heavily tested under simulated and real workload
+- is easy to use, debug and seek help from documentation
+- is performant
+- supports the most-wanted features: GC, async commit, etc.
+
+
+## Milestones
```

Comment:
Would it be better to illustrate the milestones for RawKV and TxnKV respectively?

Reply:
IMO it's unnecessary. Currently, I don't see any features in the plan that are only for RawKV. So the milestones are for TxnKV (or both). 
If there may be divergent progress in TxnKV and in RawKV (like raw_coprocessor?), then we can set separate milestones for them.